docs/plans/2026-06-13-002-feat-compound-engineering-workflow-integration-plan.md

```
@@ -0,0 +1,327 @@
+  S[CE skill in coding step] -->|Task ce-correctness-reviewer| SP{fn_spawn_agent present?}
+  SP -->|readonly: NO| F1[stripped -> spawn fails]
+  SP -->|coding: YES| RT{ce-* agent type resolvable?}
+  RT -->|not installed today: NO| F2[spawn errors / no persona]
+  RT -->|after install: YES| OK[subagent runs]
+```
+
+---
+
+## Output Structure (new/changed surfaces)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)